aviatrix/resource_aviatrix_fqdn.go

```diff
@@ -62,6 +62,7 @@ func resourceAviatrixFQDN() *schema.Resource {
 			"domain_names": {
 				Type:        schema.TypeList,
 				Optional:    true,
+				Computed:    true,
```

Comment:
Shouldn't make this computed. We should have a manage_domain_names flag similar to tgw resource

Reply:
I'm curious, why is the manage_domain_names flag a better approach than Optional and Computed?

Reply:
Making it computed will prevent managing domain_name any further on the fqdn resource. They will not be able to edit domain_names if they wish to do so.

Reply:
👍